silk/migrations/0009_add_oracle_support.py

```diff
@@ -0,0 +1,101 @@
+# Generated by Django 1.9.7 on 2018-01-10 14:14
```

Comment:
This is a very old version of Django.  Should this file be regenerated using a supported version of django?

Reply:
Also, a migration here may require the parent django application to run the migration when upgrading/downgrading django-silk which we should avoid, or if we really need to, should be very clearly messaged in the changelog.

As this migration is currently written, I think upgrading django-silk would require wiping the silk database and downgrading django-silk would not be supported (unless several manual steps are taken to reset the django-silk database which would also involve wiping the database).  This PR would need be a breaking change requiring a major version bump.